frontend/src/components/orgs-list.ts

```diff
@@ -139,6 +149,67 @@ export class OrgsList extends BtrixElement {
     `;
   }
 
+  private renderOrgProxies() {
```

Comment:
Yeah I just added it to get started, would be nice if someone with actual frontend experience could make it nice 😁

Reply:
Added a few more labels to this modal. But I think it could use a cleanup by @emma-sg or @SuaYoo, because I don't know really what I'm doing in typescript land ^^
![image](https://github.com/user-attachments/assets/de6e92cd-805f-4584-a909-9355b6e69892)